lib/index.js

```diff
@@ -58,6 +63,16 @@ module.exports = function (ops, coverage) {
       }
       that.emit('end');
     });
+
+    if(outstream){
+      var _os = fs.createWriteStream(outstream, {flags: 'a'});
```

Comment:
I was thinking a stream would be passed as the argument, not a string. Maybe support both?